ckanext/cloudstorage/cli.py

```diff
@@ -164,6 +179,91 @@ def _fix_cors(args):
         )
 
 
+def _get_unlinked_uploads(return_objects = False):
+    cs = CloudStorage()
+
+    resource_urls = (os.path.join(
```

Comment:
just need
```suggestion
    resource_urls = set(os.path.join(
```
otherwise you're creating a generator.